content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -294,6 +294,11 @@ command line arguments to `kube-apiserver`:
 * `--service-account-signing-key-file`
 * `--api-audiences` (can be omitted)
 
+You can specify the `--service-account-issuer` argument multiple times. This can be useful to
+enable a non-disruptive change of the key for an issuer.
+You must be running running Kubernetes v1.22 or later to be able to specify
+`--service-account-issuer` multiple times.
```

Comment:
I'd suggest something like this:

```
You can specify the `--service-account-issuer` argument multiple times.
This can be useful to enable a non-disruptive change of the issuer.
The first issuer specified is used when signing new tokens,
and all of the specified issuers are accepted when validating tokens presented to the API server.
You must be running running Kubernetes v1.22 or later to be able to specify
`--service-account-issuer` multiple times.
```

Reply:
Thanks, @liggitt  for looking at this.
this sounds good +1
> I'd suggest something like this:
> 
> ```
> You can specify the `--service-account-issuer` argument multiple times.
> This can be useful to enable a non-disruptive change of the issuer.
> The first issuer specified is used when signing new tokens,
> and all of the specified issuers are accepted when validating tokens presented to the API server.
> You must be running running Kubernetes v1.22 or later to be able to specify
> `--service-account-issuer` multiple times.
> ```

But this is only for --service-account-issuer flag,  should I mention about others flag or go with only  --service-account-issuer flag.

Reply:
We can copy in the CLI flag doc if that's helpful:

`--service-account-key-file`:
  * File containing PEM-encoded x509 RSA or ECDSA private or public keys, used to verify ServiceAccount tokens. The specified file can contain multiple keys, and the flag can be specified multiple times with different files.

`--service-account-issuer`:
  * When this flag is specified multiple times, the first is used to generate tokens and all are used to determine which issuers are accepted. You must be running running Kubernetes v1.22 or later to be able to specify `--service-account-issuer` multiple times.

`--api-audiences`:
  * The service account token authenticator will validate that tokens used against the API are bound to at least one of these audiences.

Reply:
yes, @liggitt  this is helpful for the readers, and I think the same thing that @sftim mentioned.
so I will update it soon.
Thanks.